### PR TITLE
Snhut 1242 group list frontend

### PR DESCRIPTION
**Pull request by sada-ajith**
https://collectiveshift.atlassian.net/browse/SNHUT-1242 has been completed
Please see the following screenrecording
[SNHUT-1242.webm](https://user-images.githubusercontent.com/109214920/184617824-3776b24b-8263-4093-96fa-1cf2cd5489b2.webm)

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-yarn pretty-quick
+#npx pretty-quick
+npx -p prettier@latest -p pretty-quick pretty-quick
```
Comment (joao-sd) [inline .husky/pre-commit:5]: Why did you change this one? Was it bugged?

File: apps/toboggan-app/src/app/group/components/group-list/group-list.component.ts [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+          name: group.name,
+          description: group.description,
+        },
+      };
+    });
+
+    this.groupList = data as TableRow[];
+  }
+
+  private getSortDirectionCoefficient(
```
Comment (joao-sd) [inline apps/toboggan-app/src/app/group/components/group-list/group-list.component.ts:119]: Not for now, but since this logic is repeated on the user table, I guess it would be better if we create a SortingService to inject in these components and avoid repetition :)

**Review by joao-sd — APPROVED** [review]
Hey Ajith, that's solid work! Great job
I left a couple of comments and there's also a conflict to solve.

Other than that, all good from my end

File: apps/toboggan-app/src/app/group/components/group-list/group-list.component.ts (hunk 2 above)
Comment (sada-ajith) [reply apps/toboggan-app/src/app/group/components/group-list/group-list.component.ts:119]: I'll update this
